akka-javasdk-testkit/src/main/java/akka/javasdk/testkit/EventSourcedResult.java

```diff
@@ -44,7 +44,7 @@ public interface EventSourcedResult<R> {
    */
   Object getUpdatedState();
 
-  boolean didEmitEvents();
+  boolean didPersistEvents();
```

Comment:
I think we should introduce such changes with a deprecation phase, even though it's "only" testkit

Reply:
done